src/web/content/static/css/letterpress.css

```diff
@@ -815,11 +820,7 @@ a:visited.badge.bg-primary
     font-family: sans-serif;
 }
 
-/**
- sms-bottom-line-container remove row for mobile? Row ads padding
- Solve the problem with internal padding, make sure that doesn't expand box.
- Lets set a rigid 100% widht or a specific viewport width.
-
-
-
-*/
+/** Forgot password screen */
+.nav-link.sms-forgot-pass-lnk {
```

Comment:
Fixes the Forgot Password link issue.

<img width="679" alt="image" src="https://github.com/searchmysite/searchmysite.net/assets/1785755/f862f930-d2fc-41eb-ae88-2465222c80e6">